pom.xml

```diff
@@ -31,6 +31,7 @@
         <module>resources</module>
         <module>releng</module>
         <module>coverage-aggregate</module>
+        <module>integration-tests</module>
```

Comment:
By adding this dependency here, the `integration-tests` module tests will be executed by default with `mvn clean install`. I doubt that we want such behaviour, as they rely also on external parameters like the HANA URL, KYMA HOST, etc. This will affect the local dev scenario, as these parameters should be provided there, or the tests should be skipped. I suggest to remove this module from here.